tools/structure/src/main.py

```diff
@@ -242,6 +249,14 @@ def run(
             structured_output_dict[SettingsKeys.METADATA] = metadata
             structured_output = json.dumps(structured_output_dict)
 
+        metrics = structured_output[SettingsKeys.METRICS]
+        # Merge dictionaries
+        new_metrics = {
```

Comment:
@Deepak-Kesavan where will the key such as `indexing` be shown in the metrics? I noticed we had `context_retrieval` though. Do add an example of the response in the PR description

Reply:
Added a sample response.

Made changes to show the key `indexing`